src/Message/ParametrizedMessageInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\Message;
+
+interface ParametrizedMessageInterface extends MessageInterface
```

Comment:
Name is a bit confusing. Parametrized means there are parameters. Here either there aren't or only one. Is there a better name?

Reply:
It was temporary named, thought about moving also `getMetadata`.

> Is there a better name?

Any suggestions?

Reply:
`IdentifyableMessageInterface`?

Reply:
Or `MessageIdInterface`? :)

Also it might be closed after introducing envelope